spec/controllers/hub/clients_controller_spec.rb

```diff
@@ -409,6 +409,22 @@
             expect(attrib.strip).to eq(message_summary.strip)
           end
         end
+
+        context "when there are clients with no current intakes (clients from previous tax years)" do
+          let!(:former_year_client) { create :client, vita_partner: organization, intake: build(:intake, :filled_out) }
+          let!(:former_year_tax_return) { create :tax_return, :intake_in_progress, client: former_year_client, year: 2020, assigned_user: assigned_user }
+
+          before do
+            # In reality this intake would be moved to the `archived_intakes_2021` table, but removing it from the DB is good enough for our purposes
+            former_year_client.intake.destroy
```

Comment:
i assume you destroyed it instead of just not creating it in the first place because of some factory rule or something